getting-started/kubernetes/customizations/imageset.md

```diff
@@ -0,0 +1,199 @@
+### Big picture
+
+Enable deploying images by docker registry digest.
+
+### Value
+
+Some deployments have strict security requirements that require deploying images by immutable digest instead of tags.
+{{site.prodname}} will not update existing tags but using an immutable digest allows specific images to be reviewed
+and verified by security teams.
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+* **ImageSet**
+
+### Concepts
+
+#### Docker repository
+
+A docker respository provides access to docker images referenced by tags or digest.
+
+#### Image tag
+
+Versioned docker images are typically referenced by a tag which is appended to an image reference. Example: `<repo>/<image>:<tag>`. Docker image tags are typically not expected be changed or updated, but this is not required or enforced.
+
+#### Image digest
+
+Docker images, when added to a docker repository, have a unique hash created that can be used to pull a specific version of an image that cannot be changed or updated.
+
+### FAQ
+
+#### What do I need to do to prepare to upgrade my cluster?
+
+When upgrading to a new patch, minor, or major release it will be necessary to create a new [ImageSet]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.ImageSet)
+that contains updated image references for the new release and named for the new release. This should be done prior
+to upgrading the cluster so when the new manifests are applied the appropriate ImageSet is available.
+
+#### Why does the Installation Resource status not include my ImageSet?
+
+The [status.imageset]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.InstallationStatus) field of
+the Installation Resource will not be updated until the `calico` component has fully been deployed. `calico` is
+fully deployed when `kubectl get tigerastatus calico` reports Available True with Progressing and Degraded as False.
+
+#### How can I tell if there is a problem with my ImageSet?
+
+If you suspect an issue with your ImageSet, check tigerastatus with `kubectl get tigerastatus`. If any components are
+degraded you can get additional information with `kubectl get tigerastatus <component-name> -o yaml`. If the digest
+provided for an image is incorrect or unable to be pulled, the tigerastatus will not directly report that information
+but you should see information that there is an issue rolling out a Deployment, Daemonset, or Job. If you suspect
+an issue with a resource rollout due to an issue with an image you will need to `get` or `describe` a specific pod
+to see details about the problem.
+
+### Before you begin
+
+- To collect the information for an ImageSet you must have a docker client configured so it can pull images from the Docker repositories where the images are stored. 
+- Once you have created an ImageSet manifest you must have permissions to apply the manifest to your cluster, so you'll need the appropriate Kubernetes configuration.
+
+### How to
```

Comment:
As previously stated, create the new How to list, and for "Create an image manifest" use the tab code:  (Manual) (Script). 

You probably will need a stem sentence to introduce the tabs. "You can create an image manifest manually or using a script."

Here is script code example:

{% tabs %}
  <label:OIDC,active:true>
<%

%>
  '<label:Google OIDC>'

<%

%>
{% endtabs %}